templates/documents/general_upload.html

```diff
@@ -33,6 +37,14 @@
 <input type="hidden" name="category_id" value="{$category_id}" />
 <input type="hidden" name="process" value="{$PROCESS}" />
 </form>
+<!-- Drag and drop uploader -->
```

Comment:
I tried that and it did not work. I did not want to dig through and try to figure out what was wrong. I took the path of least resistance.